lab_3_keywords_textrank/main.py

```diff
@@ -173,7 +186,14 @@ def extract_pairs(tokens: tuple[int, ...], window_length: int) -> Optional[tuple
     In case of corrupt input data, None is returned:
     tokens must not be empty, window lengths must be integer, window lengths cannot be less than 2.
     """
-    pass
+    if not(isinstance(window_length, int) and window_length >= 2 and tokens):
+        return None
+    pairs = set()
+    for token1 in tokens:
+        for token2 in tokens:
+            if token1 != token2 and abs(tokens.index(token1) - tokens.index(token2)) <= window_length - 1:
```

Comment:
`index` retireves the index of the first occurrence only

Reply:
fixed